src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -6155,61 +6161,6 @@ checkNullable:
             Return node.AddTrailingSyntax(b.ToList(), ERRID.ERR_Syntax)
         End Function
 
-        ''' <summary>
-        ''' Check to see if the given <paramref name="feature"/> is available with the <see cref="LanguageVersion"/>
-        ''' of the parser.  If it is not available a diagnostic will be added to the returned value.
-        ''' </summary>
-        Private Function CheckFeatureAvailability(Of TNode As VisualBasicSyntaxNode)(feature As Feature, node As TNode) As TNode
-            Return CheckFeatureAvailability(feature, node, _scanner.Options.LanguageVersion)
-        End Function
```

Comment:
If this method was retained, and implemented as `feature.IsAvailable(Options, node)`, then much of the `Parser` code could be left unchanged.